### DISCUSSION TITLE
🖐️Ajout rapide de milliers de films/series strm à la mediatheque kodi

### DISCUSSION THREAD

**Discussion by victore447** [victore447/FilmsSeriesStrmdanskodi#2] [Announcements]
![500](https://github.com/victore447/FilmsSeriesStrmdanskodi/assets/48101775/be67e846-159f-41f0-82b4-2daef7c12077)**à tous,**
[FilmsSeriesStrmdanskodi](https://github.com/victore447/FilmsSeriesStrmdanskodi) ✅

**Comment by betaa043** [comment]
Bonsoir, j'essaye de créer ma liste strm et au moment de l'extraction j'ai un message : 
C:\Users\..................\Downloads>m2strm.exe original.m3u8
File not found - original.m3u8
Configuration file not found.
Unwanted groups file not found.
Type /? for help.
J'ai pris le fichier m3u de Alldebrid et renommer en m3u8.
Pouvez vous me donner un exemple d'un fichier m3u8 de Alldebrid svp ?
Merci

**Reply by victore447** [reply]
**_Salut_** [betaa043](https://github.com/betaa043),
⛔Ces Logiciels comme  ["M2strm"](https://github.com/trix7777/m2strm#readme) etc.. ne marchent pas avec Alldebrid car ce n'est pas une playlist d'un fournisseur Iptv de Tv,Films/Séries etc... contenant des Url,Noms de films & Séries.. pour extraire ces informations indiquées dans le Repository.
☝️Les Fichiers M3u de Alldebrid ne contiennent pas de titres de films & séries (nombres de saisons),noms 
de Chaines Tv etc.. mais juste des Url d'où le blocage.

-Si l'on veux du gratuit il faut cherché un moment sur le web quand on veux pas payer auprès d'un fournisseur Iptv et il y a des sites comme [Taghdoutelive](https://www.taghdoutelive.com/search/label/IPTV?m=1).
On télécharge une liste "Iptv Xtream liste" qui contient des sources téléchargeable par "M2strm" comme ce url en exemple  👉 http://neptune.pro:8000/get.php?username=xxxxxx96&password=xxxxxa&type=m3u_plus&output=mpegts **OU** http://neptune.pro:8000/get.php?username=xxxxxx96&password=xxxxxa&type=m3u_plus&output=ts 
- **Voici un exemple en Fichier text contenant des code iptv Xtream code et des URL M3u qui contient des chaînes TV,films, 
séries etc..☝️Attention ces sources sont gratuites et ne durent pas longtemps.
Donc ne pas se fier aux dates d'expiration car ça peut s'arrêter à tout moment** 
👉 [taghdoutelive.com=STB+M3U+xtream+31-01-2024.txt](https://github.com/victore447/FilmsSeriesStrmdanskodi/files/14157726/taghdoutelive.com.STB%2BM3U%2Bxtream%2B31-01-2024.txt)


☝️Une fois le fichier m3u extrait avec le logiciel évoqué dans le présent Repo (copie dans la médiathèque Kodi etc..) 
là on n'ai obligé de faire tout manuellement & Changer les Url HS des fichiers Strm par ceux de Alldebrid quand 
ça fonctionne plus ou quand ça démarre pas dans Kodi.

☝️ **Revoir indication** [FilmsSeriesStrmdanskodi](https://github.com/victore447/FilmsSeriesStrmdanskodi) **en creant un reseau local via pc & smarphone etc.. pour copies & mise à jour rapide des url des fichiers Strm avec ceux de Alldebrid.**

-👆**Exxemple Url d'alldebrid a copier  dans les fichiers Strm qui ne durent que quelques Jours**.👉 https://k7l8m9.debrid.it/xxxxxxxxxxxxxxxxxx/Hoffa.1992.BluRay.HDLight.1080p.Multi.x264.AC3-Arcadia.mkv

👉Autrement voici quelques fournisseurs Iptv difficile à trouver,populaires et convoités en ligne et pas cher (50 à 60€ l'année) 
et sans coupures dans Kodi avec pratiquement toutes les chaînes françaises🇨🇵 etc. en replays et des milliers de Films/Séries etc..
**(ne payer que par PayPal pour le sécu du compte en banque)** ✔️
### <li>- [x] Fournisseurs Ayants Pure Iptv & Foxx Iptv: </li>
![46](https://github.com/victore447/FilmsSeriesStrmdanskodi/assets/48101775/caa9e727-800b-4827-a780-9684462ccf19)
><details>
>  <summary><b><u>Cliquer sur l'icone noir pour voir:</u></b></summary>
>
> - 🖱️ **[Consulter Site "Scamadviser" avant d'acheter](https://www.scamadviser.com/fr/accueil)**
> - 🖱️ **[Site Officel de Pure Iptv](http://www.pure-premium.com/)**
> - 🖱️ **[2eme Source Officiel Pure Iptv](https://pure-iptv.org/)**
> - 🖱️ **[King-Premium (Foxx Iptv)](https://king-premium.com/produit/vip-tv-1080-fhd-4k-ott/)**
> - 🖱️ **[Probox Iptv](https://probox-iptv.com/abonnement-iptv/)** 
> - 🖱️ **[Iptv Getline](https://www.iptv-getline.com/)**
> - 🖱️ **[Iptv Original](https://iptv-original.com/)**
